README.md

```diff
@@ -28,14 +28,222 @@ In addition, that format would include optional **signing** of the resources, wh
 
 Since the packaged "bundle" can be quite large (a game with a lot of resources or content of multiple web sites), efficient access to that content becomes important. For example, it would be often prohibitively expensive to "unpack" or somehow else pre-process such a large resource on the client device. Unpacking, for example, may require twice the space to be occupied in device's storage, which can be a problem, especially on low-end devices. We propose a optional **Content Index** structure that allows the bundle to be consumed (browsed) efficiently as is, without unpacking - by adding an index-like structure which provides direct offsets into the package.
 
-There is already a [packaging format proposal](https://w3ctag.github.io/packaging-on-the-web/) which we will base upon.
-We are proposing to improve on the spec, in particular by introducing 3 major additions:
+This is roughly based on the existing [Packaging on the Web](https://w3ctag.github.io/packaging-on-the-web/) W3C TAG proposal, but we've made significant changes:
 
-1. Hierarchical structure of the sub-packages to allow resources from multiple origins.
-2. Index of content to facilitate local resource fetching from the package.
+1. Index of content to facilitate local resource fetching from the package.
+2. Package manifests that can refer to sub-package manifests to allow resources
+   from multiple origins.
 3. Signature block, to cryptographically sign the content of the package.
+4. Removed the
+   [fragment-based URL schema](https://w3ctag.github.io/packaging-on-the-web/#fragment-identifiers)
+   from the spec as it's complex with limited use cases.
+
+Note that this is just an explainer, **not a specification**. We'll add more
+precision when we translate it to a spec.
+
+### Overall format
+
+The package is structured as follows. All "offsets" are 8-byte little-endian
+integers representing a byte offset from the start of the initial magic number.
+
+1. A magic number: `F0 9F 8C 90 F0 9F 93 A6` (🌐📦 in UTF-8).
+2. The length of the package (all bytes from the first of the initial magic
```

Comment:
It's not essential. I included it as a way to validate that the length at the end is actually referring to a package, but the other constraints here probably do that just fine. I can remove it.